blocks/library/gallery/blocks.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * any blocks.scss will be split out into a seperate blocks/build/blocks.css
```

Comment:
I believe this should be the `style.scss` instead of `blocks.scss`, and we'd have an `edit.scss` for specific editing UI.